### ISSUE TITLE
License

### ISSUE THREAD

**Issue by cstogian** [marcoregueira/EgoPDF#2]
Hello, can you specify the license for EgoPDF? I would be interested in contributing, but only with a permissive license for commercial use.
Please let me know,
Thank you.

**Comment by marcoregueira** [comment]
License added.  I will add it properly to the code  later. 

Thanks!

**Comment by marcoregueira** [comment]
Hi. Just to be sincere, egopdf source code is quite bad, it is an automated port from the original library. The parts where I modified something are easy to find, and it works, as far as I can tell. I had no resharper at the time, it would really made a difference.

I'm aware of the existence of a "manual" port of the same library to Visual Basic. It seems a lot more clear and better done.  It is VB, but if you link the assembly you will likely not notice the difference (I have to say I checked the code once, but never used it).

https://sourceforge.net/projects/fpdfnet/

Anyway, the original license of FPDF is quite permissive and it allows for any use.

**Comment by cstogian** [comment]
Hello,

Thank you for all the information. I may make some corrections anyway, and
I will commit them if I do.

Could you tell me what kind of tool you used to port the code? I would like
to port the FPDI library, if I can.

Thank you,
Christos

2016-09-16 16:21 GMT+02:00 Marco Antonio Regueira <notifications@github.com>
:

**Comment by marcoregueira** [comment]
Hi

I've used this:

http://www.asp.net/downloads/archived-v11/migration-assistants/php-to-aspnet
http://www.asp.net/downloads/archived-v11/migration-assistants/php-to-aspnet/overview


After conversion, code looked strange but it worked (mostly). I needed to
do a small number of fixes and more importantly, implement the parts that
were missing, like image support. No GD of course.

Marco

2016-09-21 13:52 GMT+02:00 cstogian <notifications@github.com>:
